app/konnect/api/ksearch/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Konnect Search API
+content_type: reference
+description: Learn how to use the Konnect Search API to search through all Konnect entities.
+beta: true
+---
+
+The {{site.konnect_short_name}} Search API allows you to search through all {{site.konnect_short_name}} entities. It is designed to cater to a wide range of search requirements, enabling you to find the information you need across different areas of the product.
+
+Here are a few example use cases where you can use the {{site.konnect_short_name}} Search API in your automation:
+* Find entities that are "not compliant". For example, you can use search to find all routes that don't have a path that starts with `/api`.
+* Select from a list of available entities. For example, if you are attaching a route to a service, you can use search to find and identify which service to attach the route to. Or when you're adding members to a team, you can search for the user.
+
+{{site.konnect_short_name}} Search includes the following features:
+
+* **Global and regional access:** The {{site.konnect_short_name}} Search API is available in global and regional locations with regional-awareness. This ensures that returned entities are relevant to their geographical location, improving response times and conforming to data residency expectations.
+* **Comprehensive response:** The responses of the search query are uniform and contain a fixed number of standard attributes: `id`, `type`, `labels/tags` and `name`. In addition, entity-specific attributes are returned in a general attributes object.
+* **Security and accessibility:** The {{site.konnect_short_name}} Search API will only return entities that the user has permissions to access. If a user is able to retrieve the entity in the [list endpoint](/konnect/api/search/latest/), then the user will be able to see the entity in the search response.
+* **Advanced query language:** The {{site.konnect_short_name}} Search API supports an advanced query language with selectors, reserved characters, and logical operators.
+
+## Supported search entities
+
+- API Product
+- API Product Version
+- Control Plane
+- Developer
+- Developer Team
+- User
+- Team
+- Report
+
+Additional entities will be added in future releases. You can view a list of all the supported entities by sending the following API request:
+
+```bash
+curl -X 'GET' \
+  'https://global.api.konghq.com/v0/search/types' \
+  -H 'accept: application/json'
```

Comment:
As we're getting ready to launch search in Konnect, we will soon be updating our search API endpoint to `v1` and remove the "beta" tag associated with it (see also [app/_data/konnect_oas_data.json](https://github.com/Kong/docs.konghq.com/pull/6655/files#diff-8f5d16af4246bd746c433ac8101739b98498f9460513e8105ffb7b5b60e53052)).
@cloudjumpercat, I'm happy to push a commit to your PR if you'd like!

Reply:
@alexgervais Hi! Thanks for letting me know! If you'd like to push a commit to this PR with your changes, that would be great!

Reply:
There's a [github action](https://github.com/Kong/docs.konghq.com/blob/main/.github/workflows/sync-konnect-oas-data.yml) that can be trigger that syncs  konnect_oas_data.json with konnect.